doc/guide.md

```diff
@@ -115,6 +116,12 @@ There are several approaches to securely operating and exposing `code-server`.
 
 **tip**: You can list the full set of `code-server` options with `code-server --help`
 
+### iPad Simple Setup
```

Comment:
Let's add a note re this app in `doc/ipad.md` and then I think we can merge!

Reply:
Ok, I see your callout to edit this file:

https://github.com/cdr/code-server/blob/v3.7.1/doc/ipad.md

I will do just that